events/system/interactionCreate.js

```diff
@@ -12,6 +12,23 @@ export default {
     }
 
     try {
+      const { userService } = interaction.services;
+      const isUser = await userService.getOneDiscordId(interaction.user.id);
+      if (!isUser) {
```

Comment:
VeryMuchANitpick: `isUser` reads like a boolean, and not the full object/row

```suggestion
      const dbUser= await userService.getOneDiscordId(interaction.user.id);
      if (!dbUser) {
```

Reply:
I might update this further when I implement to caching.